src/main/java/pl/gov/coi/pomocua/ads/configuration/AuthenticationConfig.java

```diff
@@ -8,15 +8,14 @@
 import pl.gov.coi.pomocua.ads.dev.FakeUsersRepository;
 import pl.gov.coi.pomocua.ads.users.UsersRepository;
 
+@Profile("dev")
 @Configuration
 public class AuthenticationConfig {
```

Comment:
How about naming this class `DevAuthenticationConfig` then?
Or, keep it just general `AuthenticationConfig` and keep all of the bean definitions here? I.e. keycloak + fake implementations.

Reply:
I'll rename it to DevAuthenticationConfig